WordPress/gutenberg
# Patterns Explorer: Makes the pagination items horizontally stacked on larger screens only

## What?
Fixes: #61754

## Why?
The current vertical stacking of pagination items in the Patterns Explorer doesn't make optimal use of available screen space. To address this, pagination items will now stack vertically only when the viewport is below 960px width. Beyond that, they will be horizontally aligned for better visibility.

## How?
This PR adds a media query to change the flex-direction.

## Testing Instructions
1. Go to the post editor > Options > Preferences > Accessibility
2. Enable the 'Show button text labels' preference.
3. Close the preferences modal dialog and open the main inserter > Patterns
4. Click 'Explore all patterns'
5. Scroll the modal dialog at the bottom.
6. Observe the pagination items.

## Screenshots or screencast 
https://github.com/WordPress/gutenberg/assets/77401999/48a2fed0-8f06-49b0-ac94-116db3a62468
